Login.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+
+function Login() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const response = await axios.post('/api/login', { email, password });
+      if (response.data.success) {
+        window.location.href = "/home";
+      } else {
+        alert(response.data.message);
+      }
+    } catch (error) {
```

Comment:
Avoid using alert to display errors to the user. This can lead to a poor user experience. Consider using a more user-friendly way to display errors, such as a toast notification or an error message displayed on the page.